mobile_verifier/src/boosting_oracles/assignment.rs

```diff
@@ -40,59 +60,53 @@ impl fmt::Display for Assignment {
     }
 }
 
-pub fn footfall_and_urbanization_multiplier(
-    footfall: Assignment,
-    urbanization: Assignment,
-) -> Decimal {
-    use Assignment::*;
+impl HexAssignments {
+    pub fn boosting_multiplier(&self) -> Decimal {
+        let HexAssignments {
+            footfall,
+            urbanized,
+            landtype,
+        } = self;
 
-    match (footfall, urbanization) {
-        (A, A) => dec!(1.0),
-        (A, B) => dec!(1.0),
-        (B, A) => dec!(0.75),
-        (B, B) => dec!(0.50),
-        (C, A) => dec!(0.40),
-        (C, B) => dec!(0.10),
-        (A, C) => dec!(0.00),
-        (B, C) => dec!(0.00),
-        (C, C) => dec!(0.00),
+        use Assignment::*;
+        match (footfall, landtype, urbanized) {
+            // yellow - POI ≥ 1 Urbanized
+            (A, A, A) => dec!(1.00),
+            (A, B, A) => dec!(1.00),
+            (A, C, A) => dec!(1.00),
+            // orange - POI ≥ 1 Not Urbanized
+            (A, A, B) => dec!(1.00),
+            (A, B, B) => dec!(1.00),
+            (A, C, B) => dec!(1.00),
+            // light green - Point of Interest Urbanized
+            (B, A, A) => dec!(0.70),
+            (B, B, A) => dec!(0.70),
+            (B, C, A) => dec!(0.70),
+            // dark green - Point of Interest Not Urbanized
+            (B, A, B) => dec!(0.50),
+            (B, B, B) => dec!(0.50),
+            (B, C, B) => dec!(0.50),
+            // light blue - No POI Urbanized
+            (C, A, A) => dec!(0.40),
+            (C, B, A) => dec!(0.30),
+            (C, C, A) => dec!(0.05),
+            // dark blue - No POI Not Urbanized
+            (C, A, B) => dec!(0.20),
+            (C, B, B) => dec!(0.15),
+            (C, C, B) => dec!(0.03),
+            // gray - Outside of USA
+            (_, _, C) => dec!(0.00),
+        }
     }
 }
 
-#[allow(dead_code)]
-pub fn boosting_oracles_multiplier(
-    footfall: Assignment,
-    landtype: Assignment,
-    urbanization: Assignment,
-) -> Decimal {
-    use Assignment::*;
-
-    match (footfall, landtype, urbanization) {
-        // POI ≥ 1 Urbanized
-        (A, A, A) => dec!(1.00),
-        (A, B, A) => dec!(1.00),
-        (A, C, A) => dec!(1.00),
-        // POI ≥ 1 Not Urbanized
-        (A, A, B) => dec!(1.00),
-        (A, B, B) => dec!(1.00),
-        (A, C, B) => dec!(1.00),
-        // Point of Interest Urbanized
-        (B, A, A) => dec!(0.70),
-        (B, B, A) => dec!(0.70),
-        (B, C, A) => dec!(0.70),
-        // Point of Interest Not Urbanized
-        (B, A, B) => dec!(0.50),
-        (B, B, B) => dec!(0.50),
-        (B, C, B) => dec!(0.50),
-        // No POI Urbanized
-        (C, A, A) => dec!(0.40),
-        (C, B, A) => dec!(0.30),
-        (C, C, A) => dec!(0.05),
-        // No POI Not Urbanized
-        (C, A, B) => dec!(0.20),
-        (C, B, B) => dec!(0.15),
-        (C, C, B) => dec!(0.03),
-        // Outside of USA
-        (_, _, C) => dec!(0.00),
+#[cfg(test)]
```

Comment:
would somewhere in the tests folder not be a better home for this impl?

Reply:
This method is currently only used in unit tests.
We wouldn't be able to add an implementation to `HexAssignments` without a trait, and right now I can't think of a way to justify it all.